mungegithub/mungers/nag-flake-issues.go

```diff
@@ -128,10 +129,16 @@ func (NagFlakeIssues) Munge(obj *mgh.MungeObject) {
 }
 
 // StaleComments returns a slice of stale comments
-func (NagFlakeIssues) StaleComments(obj *mgh.MungeObject, comments []*github.IssueComment) []*github.IssueComment {
+func (NagFlakeIssues) StaleComments(obj *mgh.MungeObject, issueComments []*github.IssueComment) []*github.IssueComment {
```

Comment:
I like the `Comment` refactor.  The type conversion back to `IssueComment` before return made me scratch my head for a bit.  Maybe we should change this to `StaleIssueComments` in a followup PR?